app/styles/courses/classroom-announcement-modal.sass

```diff
@@ -0,0 +1,68 @@
+@import "app/styles/mixins"
+@import "app/styles/bootstrap/variables"
+
+
+#classroom-announcement-modal
+  .modal-dialog
+    margin: 60px auto 0 auto
+    padding: 0
+    width: 746px
+    height: 520px
+    background: none
+    
+  #amazon-hoc-background
```

Comment:
The `amazon` part of this may not make sense any more 🤔

Reply:
oh, okay, i copy this from amazon-hoc thing , and i forgot to rename it 😂